deepinv/models/icnn.py

```diff
@@ -4,6 +4,139 @@
 from torch import nn
 
 
+class tiny_ICNN(nn.Module):
```

Comment:
can we call this TinyICNN to match the naming style of classes?

Reply:
It should also be good to include it somewhere in the denoisers documentation

Reply:
it's already in the list of models